sdcm/cluster_aws.py

```diff
@@ -388,11 +393,12 @@ def add_nodes(self, count, ec2_user_data='', dc_idx=0):
                 seeds = ",".join(node_private_ips)
                 if not seeds:
                     seeds = self.nodes[0].private_ip_address
-            ec2_user_data += ' --seeds %s' % seeds
+            ec2_user_data += ' --seeds %s --bootstrap true' % seeds
 
```

Comment:
Isn't there a chance to have twice "--bootstrap true"?

Reply:
Theoretically yes if we don't check before adding.

We always assign empty ec2_user_data in calling add_nodes, so it would be set in add_nodes().
I had changed code to use self._ec2_user_data first, which doesn't contain '--bootstrap' parameter. Actually it won't have repeated --bootstrap parameter.